runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -325,6 +334,17 @@ public void onChatMessage(ChatMessage event)
 		{
 			createGameTimer(SANFEW);
 		}
+
+		if (event.getMessage().startsWith("You drink some of your") && event.getMessage().contains("overload") && client.getSetting(Varbits.IN_RAID) == 1 && config.showOverload())
```

Comment:
As this message applies to both normal and raid overload, but you are specifically checking if player is in raids, at least the message check and config check should be merged to one if and then decide based on raid varbit.